chain/chain.go

```diff
@@ -12,12 +12,15 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
-var (
+const (
+	// GenesisBaseFee is the initial base fee for EIP-1559 blocks.
+	GenesisBaseFee = 1000000000
```

Comment:
Would be great if we could use scientific notation here. 
```suggestion
	GenesisBaseFee = uint64(1e9)
```
However, I think it wouldn't work, because of `types.EncodeUint64`, which doesn't handle scientific notation, but only decimal and hexadecimal numbers. 

So I'd suggest just add a comment suggesting `GenesisBaseFee` is 1 billion by default (so that we don't need to count zero every time we want to check what it genesis base fee 😄 ).

Reply:
Can we use `1_000_000_000`? much much 👓

Reply:
Done